docs/security.html

```diff
@@ -267,7 +267,9 @@
             <li>Pass the JAAS and optionally krb5 file locations as JVM parameters to each client JVM (see <a href="https://docs.oracle.com/javase/8/docs/technotes/guides/security/jgss/tutorials/KerberosReq.html">here</a> for more details):
             <pre>
     -Djava.security.krb5.conf=/etc/kafka/krb5.conf
-    -Djava.security.auth.login.config=/etc/kafka/kafka_client_jaas.conf</pre></li>
+    -Djava.security.auth.login.config=/etc/kafka/kafka_client_jaas.conf</pre>
+            For command-line utilities like kafka-console-consumer or kafka-console-producer, the above parameters can be exported via the environment variable <tt>$KAFKA_OPTS</tt> prior to execution.
```

Comment:
Thank you, I've added a note to the server configuration section too (I hadn't realised it was confusing there as well).